ComponentKit/Core/Scope/CKScopedComponentController.h

```diff
@@ -0,0 +1,17 @@
+//
+//  CKScopedComponentController.h
+//  ComponentKit
+//
+//  Created by Oliver Rickard on 3/27/17.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+@protocol CKScopedComponent;
+
+@protocol CKScopedComponentController <NSObject>
+
+- (instancetype)initWithComponent:(id<CKScopedComponent>)component;
```

Comment:
I think you made the right call. That change alone is fairly substantial, it would make reviewing these changes a bit more difficult.